types/index.d.ts

```diff
@@ -15,11 +15,11 @@ export interface CTLMatcherOptions
   container?: Element | JQuery<Element>
 }
 
-export type MatcherOptions = DTLMatcherOptions | CTLMatcherOptions
-export type ByRoleOptions = DTLByRoleOptions | CTLMatcherOptions
+export type MatcherOptions = CTLMatcherOptions | DTLMatcherOptions
+export type ByRoleOptions = CTLMatcherOptions | DTLByRoleOptions
 export type SelectorMatcherOptions =
-  | DTLSelectorMatcherOptions
   | CTLMatcherOptions
+  | DTLSelectorMatcherOptions
```

Comment:
This change was unnecessary really. I enabled a stupid lint rule as a warning that I should probably disable... 

Just did... I'll revert this bit...